tests/common/helpers/sonic_db.py

```diff
@@ -90,7 +90,7 @@ def hget_key_value(self, key, field):
             raise SonicDbKeyNotFound("Key: %s, field: %s not found in sonic-db cmd: %s" % (key, field, cmd))
         else:
             if six.PY2:
-                return result['stdout'].decode('unicode-escape')
+                return result['stdout'].decode('unicode-escape').splitlines()
```

Comment:
Why do we need `splitlines` for Python2 only? It will return a list instead of a string

Reply:
it is to support python3 str .decode method issues